packages/react-utilities/src/hooks/useForceUpdate.ts

```diff
@@ -0,0 +1,8 @@
+import { useReducer } from 'react';
+
+/**
+ * Forces a re-render, similar to `forceUpdate` in class components.
+ */
+export function useForceUpdate() {
+  return useReducer(() => ({}), {})[1];
```

Comment:
```suggestion
  return useReducer(x => x + 1, 0)[1];
```

Can we approach recommended in hooks FAQ, https://reactjs.org/docs/hooks-faq.html#is-there-something-like-forceupdate?

I know that an object _should work_, but with objects I've got strange glitches when was testing it in strict and concurrent modes.